packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -347,6 +355,17 @@ export class RichTextEditor extends FoundationElement {
                 return false;
         }
     }
+
+    // In Firefox browser, Once the editor gets focused, the blinking caret will be visible until we click format buttons (Bold, Italic ...) in the Firefox browser (Changing focus).
+    // But once any of the toolbar button is clicked, editor internally has its focus but the blinking caret disappears.
+    // As a workaround, manually triggering blur and setting focus on editor makes the blinking caret to re-appear.
+    private focusEditorInFirefox(): void {
```

Comment:
Could you update the comment to include a link to the Mozilla issue tracking this bug? That way we can periodically check if it has been fixed and maybe remove this workaround someday.